tools/test_isotovideo

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash -e
-cre="${cre:-"podman"}"
+cre="${cre:-"$(command -v podman || command -v docker ||:)"}"
+cre="${cre:?"Need either 'podman' or 'docker'"}"
```

Comment:
please don't rely on multiple redefinitions of the same variable, use a proper if/else.

Reply:
Why should we use procedural code for when we can use declarative definitions?

Reply:
> Why should we use procedural code for when we can use declarative definitions?

- Because are not being charged per character :)
- relies on specific bash knowledge
- Breaks the [Zen](https://en.wikipedia.org/wiki/Zen_of_Python)

Reply:
We use the above style in many shell scripts, e.g.
* https://github.com/os-autoinst/scripts/blob/master/backlog-check-wip-limit
* https://github.com/os-autoinst/scripts/blob/master/backlog-set-due-date
* https://github.com/os-autoinst/scripts/blob/master/openqa-advanced-retrigger-jobs
* more scripts from https://github.com/os-autoinst/scripts
* https://github.com/os-autoinst/os-autoinst/blob/master/script/os-autoinst-setup-multi-machine

So I still consider it the preferred approach. Though I am not aware of any prominent best practice guide for shell or bash scripts which would suggest a certain style.

However I don't have strong feelings about the PR that I proposed so you are welcome to decide if you want to take it as is or close unmerged or ask others for more feedback.

Reply:
> So I still consider it the preferred approach.

I also like it :-)

(You could get rid of the outer quotes, though.)

Reply:
The outer quotes are suggested by shellcheck AFAIK

Reply:
@okurz I see your point; still, make it an if-else please;

Reply:
Sorry, I can't